PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-patJetsUpdated = cms.EDProducer("PATJetUpdater",
+updatedPatJets = cms.EDProducer("PATJetUpdater",
```

Comment:
this change of name without changing of content, is also not necessary... and introduce troubles to physics analysis using this module.